packages/core/src/js/sdk.tsx

```diff
@@ -62,6 +63,40 @@ export function init(passedOptions: ReactNativeOptions): void {
     enableSyncToNative(getIsolationScope());
   }
 
+  const getURLFromDSN = (dsn: string | null): string | undefined => {
+    if (!dsn) {
+      return undefined;
+    }
+    const url = new URL(dsn);
```

Comment:
Let's wrap the URL with try catch, RN has it's own specific implementation and users often change it with polyfills.

Reply:
Good catch @krystofwoldrich 🙇 
Updated with 41ac85a826d11623f6f7ba22793c7f3a736babad